### DISCUSSION TITLE
Module "blowfish" not compatible warning in Hugo 0.118.2

### DISCUSSION THREAD

**Discussion by CRISHTANG** [nunocoracao/blowfish#941] [Q&A] [answered]
I'm new to both Hugo and blowfish, and tried to look into what to do in the case of this error but found nothing of help.  I installed the latest Hugo (v0.118.2) and installed blowfish as a git submodule as recommended in the documentation.  Upon running hugo server it produces the warning:

> WARN  Module "blowfish" is not compatible with this Hugo version; run "hugo mod graph" for more information.

Looking at the config.toml in the blowfish theme's root directory, it contains:

```
[module]
  [module.hugoVersion]
    extended = true
    min = "0.87.0"
    max = "0.117.0"
```

So I would assume since my Hugo version is 0.118.2 that it is out of bounds and throwing the error for that reason.  My question is what would be the best course of action?  Downgrade my Hugo version to 0.117.0?  Or will Blowfish still function properly under 0.118.2?  Or does blowfish need to be updated to run under it?

I also noticed that in the blowfish documentation that it states to "delete the config.toml file that was generated by Hugo." in the root folder.  Hugo did not generate a config.toml in my root directory, but it did generate a hugo.toml which is the file that the official Hugo tutorials says to append with my chosen theme (which the blowfish docs does not mention).  Not sure if that matters or if its related to it being under a newer Hugo version, but coming in cold its a bit confusing so I thought I should mention it.

Looking forward to working with this theme, thanks in advance for any advice!

**Comment by nunocoracao** [comment, marked as answer]
Hello @CRISHTANG this warning shouldn't cause any issues with using the theme. 
I keep a max version in that file so that I need to test and update the theme - sometimes Hugo introduces breaking changes
You can follow the issue here - https://github.com/nunocoracao/blowfish/issues/945
Should be fixed in the next couple of hours
